share/gpodder/extensions/podverse.py

```diff
@@ -0,0 +1,72 @@
+# -*- coding: utf-8 -*-
+# Searches podverse (podverse.fm) database for podcasts
+
+# (c) 2024 Eric Le Lay <elelay.fr:contact>
+# Released under the same license terms as gPodder itself.
+
+# Inspired by gpodder-core plugin "podverse", by kirbylife <hola@kirbylife.dev>
+# https://github.com/gpodder/gpodder-core/blob/master/src/gpodder/plugins/podverse.py
+
+import logging
+from urllib.parse import quote_plus
+
+import gpodder
+from gpodder.directory import PROVIDERS, DirectoryEntry, Provider
+from gpodder.util import urlopen
+
+logger = logging.getLogger(__name__)
+
+_ = gpodder.gettext
+
+__title__ = _('Search Podverse')
+__description__ = _('Search podverse podcast index')
+__authors__ = 'Eric Le Lay <elelay.fr:contact>'
+__doc__ = 'https://gpodder.github.io/docs/extensions/podverse.html'
+
+
+class PodverseDirectoryProvider(Provider):
+    def __init__(self):
+        self.name = _('Podverse search')
+        self.kind = Provider.PROVIDER_SEARCH
+        self.icon = None  # TODO: there is no way for a provider to register a custom icon
```

Comment:
Why can't the icon be placed in `share/gpodder/images/` with the others? I understand it wouldn't work for user installed extensions, and the code should be fixed to also search `~/.local/share/gpodder/images/` or `~/gPodder/Extensions/`.

Reply:
done